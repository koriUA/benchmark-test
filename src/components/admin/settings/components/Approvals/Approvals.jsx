import React from 'react';

const Approvals = () => <div>Here goes content for Approvals</div>;

export default Approvals;
