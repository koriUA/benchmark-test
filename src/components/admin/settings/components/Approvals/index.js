import Approvals from './Approvals';

export default Approvals;
