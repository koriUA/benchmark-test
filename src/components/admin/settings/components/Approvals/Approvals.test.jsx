import React from 'react';
import { mount } from 'enzyme';

import Approvals from './Approvals';

describe('Approvals', () => {
  test('should render the Approvals', () => {
    const component = mount(<Approvals />);
    expect(component).toMatchSnapshot();
  });
});
