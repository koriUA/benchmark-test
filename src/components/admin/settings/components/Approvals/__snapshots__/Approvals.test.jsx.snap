// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Approvals should render the Approvals 1`] = `
<Approvals>
  <div>
    Here goes content for Approvals
  </div>
</Approvals>
`;
